### PR TITLE
[ACM-2.10] ACM#9991: Install a hosted cluster by using mirror registries

### PR DESCRIPTION
**Pull request by xenolinux**
https://issues.redhat.com/browse/ACM-9991

Peer review approval is present in https://github.com/stolostron/rhacm-docs/pull/6397

### CODE REVIEW

File: clusters/hosted_control_planes/create_cluster_bm.adoc [hunk 1]
```diff
@@ -11,23 +11,23 @@ oc create ns <hosted-cluster-namespace>-<hosted-cluster-name>
 ----
 
 +
-Replace `<hosted-cluster-namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted-cluster-name>` with your hosted cluster name.
+Replace `<hosted_cluster_namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted-cluster-name>` with your hosted cluster name.
```
Comment (CrystalChun) [inline clusters/hosted_control_planes/create_cluster_bm.adoc]:
```suggestion
Replace `<hosted_cluster_namespace>` with your hosted cluster namespace name, for example, `clusters`. Replace `<hosted_cluster_name>` with your hosted cluster name.
```

File: clusters/hosted_control_planes/create_cluster_bm.adoc [hunk 2]
```diff
@@ -102,6 +102,56 @@ The **Hosted cluster** view is displayed.
...
+
++
+<1> Specify the name of your hosted cluster, for instance, `example`.
+<2> Specify the path to your pull secret, for example, `/user/name/pullsecret`.
+<3> Specify your hosted control plane namespace, for example, `clusters-example`. Ensure that agents are available in this namespace by using the `oc get agent -n <hosted-control-plane-namespace>` command.
+<4> Specify your base domain, for example, `krnl.es`.
+<5> Specify the `icsp.yaml` file that defines ICSP and your mirror registries.
+<6> Specify the path to your SSH public key. The default file path is `~/.ssh/id_rsa.pub`.
+<7> Specify your hosted cluster namespace.
+<8> Specify the supported {ocp-short} version that you want to use, for example, `4.14.0-x86_64`.
```
Comment (CrystalChun) [inline clusters/hosted_control_planes/create_cluster_bm.adoc]: If using ICSPs and mirror registries, they'll need this to be in digest format too. Could we have a similar instruction as https://github.com/stolostron/rhacm-docs/pull/6392/files#diff-b06c54184e168a5cf37dfb3f39cd0c506fb6f7147e2cda15d820a0bf570d15ddR41

Comment (xenolinux) [reply clusters/hosted_control_planes/create_cluster_bm.adoc]: Done. Thanks!

File: clusters/hosted_control_planes/create_cluster_bm.adoc (hunk 1 above)
Comment (xenolinux) [reply clusters/hosted_control_planes/create_cluster_bm.adoc]: Nice catch!

**Review by CrystalChun — APPROVED** [review]
/lgtm

**Pull request merged**